src/transit/tests.py

```diff
@@ -6,6 +6,21 @@
 BASE_DEV_URL = environ["TRANSIT_DEV_BACKEND_URL"]
 BASE_PROD_URL = environ["TRANSIT_BACKEND_URL"]
 
+# We want to verify that /allstops returns a list of type 'busStop' only
+def allstops_returns_busStops(r):
+    response = r.json()
+    # Make sure response was successful
+    if not response["success"]:
+        return False
+
+    # Iterate over stops
+    for bus_stop in response["data"]:
+        # Check that the BusStop object is properly decoded
+        if "type" not in bus_stop or bus_stop["type"] != "busStop":
+            return False
+    return True
+
+
```

Comment:
probably delete this newline?

Reply:
should i delete the extra newlines for all the other functions (i noticed 2 newlines between all the older fn's)